ambry-cloud/src/main/java/com/github/ambry/cloud/azure/AzureCloudDestinationSync.java

```diff
@@ -87,7 +95,8 @@ public class AzureCloudDestinationSync implements CloudDestination {
    * @param metricRegistry
    * @param clusterMap
    */
-  public AzureCloudDestinationSync(VerifiableProperties verifiableProperties, MetricRegistry metricRegistry, ClusterMap clusterMap)
+  public AzureCloudDestinationSync(VerifiableProperties verifiableProperties, MetricRegistry metricRegistry,
```

Comment:
javadoc as well, please review the javadoc for the whole PR.

Reply:
done thank you